runtime/src/runtime_api.rs

```diff
@@ -60,71 +55,18 @@ pub type BlockId = generic::BlockId<Block>;
 pub type UncheckedExtrinsic = generic::UncheckedExtrinsic<AccountId, Call, Signature, SignedExtra>;
 
 // Default Executive type without the RuntimeUpgrade
-// pub type Executive =
-//     frame_executive::Executive<Runtime, Block, frame_system::ChainContext<Runtime>, Runtime, AllModules>;
-
-// Alias for the builder working group
-pub(crate) type OperationsWorkingGroup<T> =
-    working_group::Module<T, OperationsWorkingGroupInstance>;
-
-// Alias for the gateway working group
-pub(crate) type GatewayWorkingGroup<T> = working_group::Module<T, GatewayWorkingGroupInstance>;
-
-// Alias for the storage working group
-pub(crate) type StorageWorkingGroup<T> = working_group::Module<T, StorageWorkingGroupInstance>;
-
-// Alias for the content working group
-pub(crate) type ContentDirectoryWorkingGroup<T> =
-    working_group::Module<T, ContentDirectoryWorkingGroupInstance>;
+// pub type Executive = frame_executive::Executive<
```

Comment:
Shouldn't we toggle the `Executive` comments here? We have commented `Executive` without the `OnRuntimeUpgrade` handler.

Reply:
I did that initially, and it required removing a few more unused imports. So I decided to keep empty implementation and left these comments (I think I just moved it closer and did some rust-fmt on it)
It might make more sense to just remove the commented code and always use the CustomOnRuntimeUpgrade implementation.

Reply:
What do you think ? dac89e7bbcf4ab6aba46ae6913eea83e633e4a32